projects/packages/search/src/instant-search/components/search-result-expanded.jsx

```diff
@@ -77,7 +77,7 @@ export default function SearchResultExpanded( props ) {
 					<div className="jetpack-instant-search__search-result-expanded__image-container">
 						{ firstImage ? (
 							<PhotonImage
-								alt={ fields[ 'title.default' ] }
+								alt={ fields[ 'image.alt_text' ] }
```

Comment:
I thought about keeping `title.default` as a fallback here, but it's complicated by the fact that `alt=""` is [valid alt text](https://www.w3.org/WAI/tutorials/images/decorative/). 

Feels best to rely on the alt text from the API in all cases.